### DISCUSSION TITLE
To Get Past the Gatekeeper

### DISCUSSION THREAD

**Discussion by thefatsasquatch** [community/community#49840] [GitHub Education]
I'm at a loss here, and I need some help.

My college does not date IDs. I have uploaded pictures of my tuition bill and schedule.

How can we get human eyes on the issue?

![Spring 23 Receipt](https://user-images.githubusercontent.com/117497007/224525537-c223b32b-60df-4081-9d04-857446a5a66b.jpg)
<img width="1151" alt="Spring 23 Schedule" src="https://user-images.githubusercontent.com/117497007/224525540-833b592c-c0b9-49f1-9451-64ebee4148b5.png">
![ID](https://user-images.githubusercontent.com/117497007/224525564-53b10ef1-4d30-4711-8a03-fdc8aef62048.jpg)

**Comment by Ntshangase** [comment]
Hello, I'm assuming you may be trying to apply for the GitHub Global Campus, I would suggest logging an issue directly with GitHub at: https://support.github.com/contact?tags=hubberfy_account%2Crr-accounts

**Reply by samus-aran** [reply]
Hi @Ntshangase, thanks for trying to assist @thefatsasquatch but we don't direct users to support if they are requesting for **individual application reviews** or **application statuses**. We recently made an[ announcement on the support available for GitHub Education](https://github.com/orgs/community/discussions/49366). We're here to assist ⭐

**Comment by samus-aran** [comment]
Hi @thefatsasquatch 👋 

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

We do accept non dated Student/Faculty IDs but we need to understand what other factors are at play by seeing the rejection reasons.
